RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -203,15 +206,21 @@
 fastSim.toModify(detachedQuadStepTracks,TTRHBuilder = 'WithoutRefit')
 
 # TRACK SELECTION AND QUALITY FLAG SETTING.
-from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
-from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-detachedQuadStep = TrackLwtnnClassifier.clone(
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierDetached_cfi import *
+detachedQuadStep = TrackMVAClassifierDetached.clone(
+    mva = dict(GBRForestLabel = 'MVASelectorDetachedQuadStep_Phase1'),
     src = 'detachedQuadStepTracks',
-    qualityCuts = [-0.6, 0.05, 0.7],
+    qualityCuts = [-0.5,0.0,0.5]
 )
 fastSim.toModify(detachedQuadStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
please move this line after trackdnn.toReplace (or even after pp_on_AA).
This is a bit excessive, but it will cover all cases where fastsim might be added.

Reply:
Moved to after pp_on_AA. Also changed L223 highBetaStar_2018.toReplaceWith -> .toModify, as it was before.